BusyTiming/ViewControllers/TimerViewController.swift

```diff
@@ -35,7 +61,7 @@ class TimerViewController: UIViewController {
     }
     
     @IBAction func  addActivityButtonPressed(_ sender: UIButton){
-        timerModel.lastTimePoint = Date().timeIntervalSince1970
+        timerVM.timerModel.lastTimePoint = Date().timeIntervalSince1970
```

Comment:
the view should not know anything about the model, just the view model. furthermore, it should not assign any values even to the view model (except the values from text fields, pickers, switches, or other changeable UI elements). all changes to the view model should be moved to the view model. the view should just notify the view model about an event.